velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -66,6 +87,27 @@ class PlanBuilder {
       const std::shared_ptr<core::InsertTableHandle>& insertHandle,
       const std::string& rowCountColumnName = "rowCount");
 
+  /// Adds an AggregationNode representing partial aggregation with the
+  /// specified grouping keys, aggregates and optional masks.
+  ///
+  /// Grouping keys are specified using zero-based indices into the input
+  /// columns.
+  ///
+  /// Aggregates are specified as function calls over unmodified input columns,
+  /// e.g. sum(a), avg(b), min(c). SQL statement AS can be used to specify names
+  /// for the aggregation result columns. In the absence of AS statement, result
+  /// columns are named a0, a1, a2, etc.
+  ///
+  /// For example,
+  ///
+  ///     partialAggregation({}, {"min(a) AS min_a", "max(b)"})
+  ///
+  /// will produce output columns min_a and a1, while
+  ///
+  ///     partialAggregation({0, 1}, {"min(a) AS min_a", "max(b)"})
+  ///
+  /// will produce output columns k1, k2, min_a and a1, assuming the names of
```

Comment:
why is `max(b)` alias `a1` and not `a3`?

Reply:
That's because only aggregate columns get auto-generated names. The name is a<N>, where N is the ordinal position of the aggregate starting with zero.